direct/src/distributed/direct.dc

```diff
@@ -3,14 +3,13 @@
 // your own project (or load it from the direct source tree) and build
 // on it with your own dc file for your own classes.
 
-keyword broadcast;
-keyword ram;
-keyword p2p;
```

Comment:
Removing the other forward-declarations of keywords is fine, however `p2p` is not a builtin keyword and carries specific meaning with CMU's `ServerRepository`.